### ISSUE TITLE
Unable to add single honeypot monitoring - LogicApp issue HoneyTokensAdd2WLKV

### ISSUE THREAD

**Issue by Ryk-Send** [Azure/Azure-Sentinel#10694]
**Describe the bug**
Clicking on the ling to monitor a honeypot item I get the following response:
{
    "error": {
        "code": "NoResponse",
        "message": "The server did not receive a response from an upstream server. Request tracking id '0858482304889453597723302548xxxxx."
    }
}

When I dig deeper I can see in the following LogicApp: "HoneyTokensAdd2WLKV" there is an error in the "Watchlists - Add a new watchlist item" step and the error is: "Unable to initialize operation details for swagger based operation - Watchlists_-_Add_a_new_watchlist_item. Error details - The API 'azuresentinel-honeytokens' could not be found."

![image](https://github.com/Azure/Azure-Sentinel/assets/139942348/fb3b5ca3-06b6-4281-ad06-4ccf76c1a274)

**Comment by v-rusraut** [comment]
Hi @Ryk-Send, Thanks for flagging this issue, we will investigate this issue and get back to you with some updates by 05 July 2024. Thanks!

**Comment by v-sudkharat** [comment]
Hi @Ryk-Send, Waiting for your response on above comment. Thanks!

**Comment by Ryk-Send** [comment]
> Hi @Ryk-Send, We are checking on this issue, Meantime sharing a document which may help for this issue:- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/HoneyTokens/README.md
> 
> Thanks!

I followed these steps to the letter and still the same issue.

**Comment by v-sudkharat** [comment]
@Ryk-Send, thanks for response, will check and get back to you on this issue. Thanks!

**Comment by v-sudkharat** [comment]
Hi @Ryk-Send, Waiting for your response on above comment. Thanks!

**Comment by v-sudkharat** [comment]
Hi @Ryk-Send, Waiting for your response on above comment. Thanks!

**Comment by v-sudkharat** [comment]
Hi @Ryk-Send, Gentle Reminder: We are waiting for your response on this issue. If you still need to keep this issue active, please respond to it in the next 2 days. If we don't receive a response by 07-08-2024 date, we will be closing this issue. 		
Thanks!

**Comment by Ryk-Send** [comment]
Sorry for that. I have moved on as this was not working and the way to get it working just too technical for me.

I think I will just uninstall the solution. In your screenshot above the one of the fields there asks for JSON of the watchlist items. I just dont have the time to work out what needs to go in there. That is why I opted for the solution instead of building it myself.

When it is stable in the future then I will give it another go.

Thanks again for your great work here man, appreciate it.
